ably/rest_client_test.go

```diff
@@ -187,6 +187,11 @@ func TestRestClient(t *testing.T) {
 		}
 		res.Body.Close()
 
+		// Artificial delay to provide sandbox time to populate stats data that
+		// was just POST. This does not negatively impact test timings because
+		// tests are run in parallel.
+		<-time.After(time.Second * 5)
```

Comment:
Rather than a 5 second sleep, we should call `client.Stats` in a loop with a small sleep that returns when there are some stats or when a timeout is hit. Let me know if you'd like me to provide an example of that.

Reply:
That sounds like a good solution to me!

Reply:
Thanks both. 👍